com.github.slgobinath.SafeEyes.yaml

```diff
@@ -0,0 +1,99 @@
+id: com.github.slgobinath.SafeEyes
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+command: safeeyes
+
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--system-talk-name=org.freedesktop.DBus" # for tray icon
```

Comment:
I don't think this VERY BROAD permission is neceary.

Reply:
Is it ok to use `org.freedesktop.StatusNotifierItem.Menu` ?